packages/runner-ct/src/app/RunnerCt.tsx

```diff
@@ -53,13 +63,18 @@ const App: React.FC<AppProps> = observer(
     const { state, eventManager, config } = props
 
     const [activeIndex, setActiveIndex] = React.useState<number>(0)
-    const [search, setSearch] = React.useState('')
     const headerRef = React.useRef(null)
 
-    const runSpec = React.useCallback((file: FileNode) => {
+    function runSpec (file: FileNode) {
       setActiveIndex(0)
-      state.setSingleSpec(state.specs.find((spec) => spec.absolute.includes(file.absolute)))
-    }, [state])
+      const selectedSpec = props.state.specs.find((spec) => spec.absolute.includes(file.relative))
+
+      if (!selectedSpec) {
+        throw Error(`Could not find spec matching ${file.relative}.`)
+      }
+
+      state.setSingleSpec(selectedSpec)
+    }
```

Comment:
Merge issue? Seems odd that you would change it back from using `useCallback`

Reply:
merge issue, will fix